services/QuillLMS/spec/workers/sync_sales_form_submission_to_vitally_worker_spec.rb

```diff
@@ -84,6 +84,34 @@
 
       subject.create_vitally_user_if_none_exists
     end
+
+    it 'should update a Vitally user if with two different districts if the users school already has a different district attached' do
```

Comment:
The `it` description of both of these tests is a bit confusing, but the tests themselves look good.

Reply:
Yeah - if you find yourself adding more 1 or more "if" conditions within an `it` description, it's probably useful to express the "if" conditions as `context` blocks.